crates/api_models/src/disputes.rs

```diff
@@ -108,41 +109,67 @@ pub struct DisputeEvidenceBlock {
     pub file_metadata_response: files::FileMetadataResponse,
 }
 
-#[derive(Clone, Debug, Deserialize, ToSchema, Serialize)]
+#[derive(Clone, Debug, Deserialize, Serialize, ToSchema)]
 #[serde(deny_unknown_fields)]
-pub struct DisputeListConstraints {
-    /// limit on the number of objects to return
-    pub limit: Option<i64>,
+pub struct DisputeListGetConstraints {
+    /// The identifier for dispute
+    pub dispute_id: Option<String>,
```

Comment:
Why do we need `dispute_id` in DisputeListGetConstraints? We can use Dispute retrieve for point query right?

Reply:
Its a good to have: 
-  Users will be able to search for a specific dispute by its ID and see the result directly in the list and then it can be retrieved. If the dispute doesn't exist, the list page should display a message indicating that the dispute is not present.
- Users will have the ability to search for disputes one by one, without needing to repeatedly navigate back to the retrieve page. They should be able to see disputes directly in list (if it exists), making the process more user-friendly and efficient.